bin/release-to-branch.sh

```diff
@@ -20,9 +20,7 @@ fi
 git checkout -b $BRANCH_NAME
 
 # Build the package as normal
-export DMTASK='preparing'
 npm run build
-unset DMTASK
```

Comment:
I assume this was doing nothing useful?

Reply:
It was originally changing where the compiled javascript ended up; it would go to a different location if DMTASK had the value of 'preparing'.

However this makes building locally annoying because you have to remember to change DMTASK based on what you want to do, and it seems to me that copying an extra file is much easier to deal with than the extra code complexity